### PR TITLE
ci: fix medic benchmark workflow

### PR DESCRIPTION
**Pull request by lenaschoenburg**
Fixes various small issues in the medic benchmark workflow.

### CODE REVIEW

File: .github/workflows/medic-benchmarks.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ jobs:
       name: medic-cw-${{ needs.benchmark-data.outputs.cw }}-${{ needs.benchmark-data.outputs.ref }}-benchmark
       cluster: zeebe-cluster
       cluster-region: europe-west1-b
-      ref: ${{ steps.data.outputs.ref }}
+      ref: ${{ needs.benchmark-data.outputs.ref }}
```
[the remark this answers is not in the document]
Comment (oleschoenburg) [reply .github/workflows/medic-benchmarks.yml:53]: Sorry, yeah :+1: Went into a different commit that I then dropped. Fixed now.

**Comment by saig0** [comment]
@oleschoenburg the previous issue is fixed. :+1: 

Now, there is an issue when deleting previous benchmarks: https://github.com/camunda/zeebe/actions/runs/3618091736/jobs/6097549980

**Comment by saig0** [comment]
@oleschoenburg nice. :+1: 

It seems there is one more issue with extracting the calendar week: https://github.com/camunda/zeebe/actions/runs/3618091736/jobs/6097549806

File: .github/workflows/medic-benchmarks.yml [hunk 2]
```diff
@@ -67,7 +67,7 @@ jobs:
       cluster: zeebe-cluster
       cluster-region: europe-west1-b
       ref: ${{ needs.benchmark-data.outputs.ref }}
-      bechmark-load: >
+      benchmark-load: >
```
Comment (saig0) [inline .github/workflows/medic-benchmarks.yml:70]: :thinking: We have the same typo in the benchmark workflow: https://github.com/camunda/zeebe/blob/main/.github/workflows/benchmark.yml#L20

We need to rename both.

Comment (oleschoenburg) [reply .github/workflows/medic-benchmarks.yml:70]: Looks like it works now: https://github.com/camunda/zeebe/actions/runs/3618326752 :sweat_smile:

File: .github/workflows/medic-benchmarks.yml [hunk 3]
```diff
@@ -27,12 +27,15 @@ jobs:
...
-          kubectl delete ${{ steps.data.normal }}-mixed
-          kubectl delete ${{ steps.data.normal }}-latency
+          kubectl delete ${{ steps.data.outputs.normal }}
+          kubectl delete ${{ steps.data.outputs.normal }}-mixed
+          kubectl delete ${{ steps.data.outputs.normal }}-latency
   benchmark-data:
     name: Collect benchmark data
     runs-on: ubuntu-latest
+    outputs:
+      ref: ${{ steps.data.outputs.ref }}
```
Comment (saig0) [inline .github/workflows/medic-benchmarks.yml:37]: :x: There seems to be an issue with the `ref`. 

See https://github.com/camunda/zeebe/actions/runs/3618326752/jobs/6098054983#step:2:6
It passed an empty `ref` to the benchmark creation: https://github.com/camunda/zeebe/actions/runs/3618326752/jobs/6098090182

@oleschoenburg please have a look.

**Review by saig0 — CHANGES_REQUESTED** [review]
@oleschoenburg nice improvements. :rocket: 

We are close. But I found one more issue. Please have a look at my comment.

**Comment by saig0** [comment]
@oleschoenburg the last commit fixed the issue. But now, the step "Deploy benchmark cluster" failed: https://github.com/camunda/zeebe/actions/runs/3618738541/jobs/6099935870. :see_no_evil:

**Comment by oleschoenburg** [comment]
That were a lot of small issues, sorry about that! I think it's working now: https://github.com/camunda/zeebe/actions/runs/3619314666

**Review by saig0 — APPROVED** [review]
@oleschoenburg awesome. :partying_face: 

It works like a charm. :star_struck:

**Pull request merged**